.github/workflows/bench.yml

```diff
@@ -63,7 +75,23 @@ jobs:
         run: |
           sudo apt-get update && sudo apt-get install -y python3-pip
           pip3 install jtbl
-      - name: Parse inputs
+      - name: Set env
+        run: |
+          # Default benchmark settings optimized for Aptos, can be overwritten with `env` input
+          echo "RUST_LOG=debug" | tee -a $GITHUB_ENV
+          echo "RUSTFLAGS='-C target-cpu=native --cfg tokio_unstable -C opt-level=3'" | tee -a $GITHUB_ENV
+          echo "SHARD_SIZE=4194304" | tee -a $GITHUB_ENV
+          echo "SHARD_BATCH_SIZE=0" | tee -a $GITHUB_ENV
```

Comment:
This is not possible with the current head of the `dev` branch of Sphinx. As @storojs72 's  work pointed out that we should set it to `16` and that's the default value, we should remove it.